code/modules/client/preference_setup/loadout/items/accessories.dm

```diff
@@ -701,3 +701,19 @@
 	display_name = "visegradi patterned sweater"
 	path = /obj/item/clothing/accessory/sweater/visegradi
 	flags = GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/sec_mask
+	display_name = "corporate protective mask selection"
+	description = "A generic camera-assisted mask issued to members of SCC security teams."
+	path = /obj/item/clothing/accessory/sec_mask
+	flags = 0
+	allowed_roles = list("Security Officer", "Head of Security", "Warden", "Security Cadet", "Investigator", "Security Personnel")
```

Comment:
Remove this. Loadout should not have job equipment, and especially not armored gear and/or stuff that hides your face.

Reply:
Should be in the sec armory instead.

Reply:
At the request of Noble who commissioned these, I moved them to the security lockers rather than the armoury, to make them more accessible while still not being in the loadout.

Reply:
~~This is still in the loadout.~~ removed already

Should be in the armory, not lockers.

When tactical coifs were being added, they were made to not hide the user's face, and were put in the armory anyways. And this protective mask is even more tacticool than these coifs, so it should be in the armory just the same.

Reply:
Do not put the hoplan masks on horizon, cause "the hoplan" is golden deep military, so why would SCC stock up its security department with hoplan gear?

Reply:
Done and done